src/main/java/ch/uzh/ifi/hase/soprafs22/service/UserService.java

```diff
@@ -86,12 +87,32 @@ private long getAgeInMilliSeconds(Date birthday) {
     }
 
     private int getAge(Date birthday) {
-        Calendar c = Calendar.getInstance();
-        c.setTime(new Date());
-        int currentYear = c.get(Calendar.YEAR);
-        c.setTime(birthday);
-        int birthYear = c.get(Calendar.YEAR);
-        return currentYear - birthYear;
+        int years;
+        int months;
+        Calendar birthDayMili = Calendar.getInstance();
+        birthDayMili.setTimeInMillis(birthday.getTime());
```

Comment:
we have a method getAgeInMilliSeconds

Reply:
yes, but that method is just a lazy class that implements an already existing method i think?

Reply:
I don't know what you mean

Reply:
getAgeinMilliSeconds does the same thing as an already existing method

Reply:
If so, you could remove that method and use the already existing method?

Reply:
also irrelevant now